src/quo/components/list_items/token_network/view.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo.components.list-items.token-network.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.list-items.preview-list.view :as preview-list]
+    [quo.components.list-items.token-network.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.common :as common]
+    [quo.foundations.resources :as resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn- info
+  [{:keys [token networks]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (resources/get-token token)
+     :style  style/token-image}]
+   [rn/view
+    [text/text {:weight :semi-bold} (common/token-label token)]
```

Comment:
@J-Son89 Is it ok use - `common/token-label`?
```
(def token-label
  {:eth "Ethereum"
   :snt "Status"
   :dai "Dai"})
   ```
   Implementation does not look complete

Reply:
discussed off github, decided to go with string param